charts/tekton-pipelines/templates/buildpacks/tasks/_buildpack.yaml

```diff
@@ -159,6 +169,11 @@ spec:
 
         aws ecr get-login-password --region $AWS_REGION | docker login -u AWS --password-stdin $(params.docker_registry)
 
+        if [ "$(params.add_tag_latest)" == "true" ]
+        then
+          ADDITIONAL_TAG="-tag=$(params.docker_registry_repository):latest"
+        fi 
+
```

Comment:
let's adjust this functionality so we can only add it if we're in the **prod** environment, so we don't accidentally push this tag from dev or staging envs.

I suggest using **params.environment** for the check

Reply:
Also adjust the same behavior to kaniko task. So the logic is the same between kaniko and buildpacks.

Reply:
Hi Dima! I will adjust for the kaniko, but we need this behaviour on all three envs (dev, staging, prod) on the xrayimg project. Moreover, all these changes made because we need this behaviour on dev env

Reply:
ok, do we have dev, staging, prod ECR repos isolated by prefix i.e.
ecr://domain/backend/dev
ecr://domain/backend/staing
ecr://domain/backend/prod ?

Reply:
> ok, do we have dev, staging, prod ECR repos isolated by prefix i.e. ecr://domain/backend/dev ecr://domain/backend/staing ecr://domain/backend/prod ?

Yes, of course. What do you mean? 

> Also adjust the same behavior to kaniko task. So the logic is the same between kaniko and buildpacks.

I decided do not adjust kaniko pipelines: we already have an option to pass additional build arguments. Added how to tag image in docs.